src/sphinx/4.0.x/features/scaladebugger/index.rst

```diff
@@ -144,28 +134,26 @@ Using the Scala debugger for a remote debugging session is done by selecting one
 .. image:: images/remotedebugging-01.png
    :alt: Scala remote debugging configuration
 
-Status
-------
 
-* Smart stepping over/in/out anonymous functions (for comprehension, collection methods, ...).
-* Resume/terminate launch action.
-* The structure of the Scala implementation of the Eclipse debug model.
-* Improved label and value display.
-* No good icons for the model elements.
-* No caching of any data.
-* No drop to frame.
-* Need to define and spec the exact behavior of the smart step in/over/out.
+Drop to frame |new| [since 4.0]
+-------------------------------
 
-known issues
-............
+**To be described in next PR**
+
+Hot code replace |new| [since 4.1]
+----------------------------------
+
+**To be described in next PR**
 
+Expression evaluator |new| [added in 4.0, updated in 4.1]
+---------------------------------------------------------
 
-TODO
-....
+**To be described in next PR**
 
-*may not be exhaustive or up-to-date, not really ordered*
+Further improvements and unsupported features
```

Comment:
I don't think we should have this section at all. If there is a problem, the ticket tracker should keep it.